tests/wallet/clawback/test_clawback_lifecycle.py

```diff
@@ -196,7 +199,9 @@ async def test_clawback_spends(self, cost_logger: CostLogger) -> None:
                 cost_log_msg="Create Second Clawback",
             )
 
-            new_cb_coin = (await sim_client.get_coin_records_by_puzzle_hash(cb_puz_hash))[1].coin
+            new_cb_coin = (await sim_client.get_coin_records_by_puzzle_hash(cb_puz_hash, include_spent_coins=False))[
```

Comment:
an alternative way to do this, that also ensures we get exactly 1 coin back, would be:
```
[new_cb_coin] = (await sim_client.get_coin_records_by_puzzle_hash(cb_puz_hash, include_spent_coins=False))
```

Reply:
thanks